papaparse.js

```diff
@@ -615,11 +615,11 @@
 				this._chunkError();
 			else
 				this._start += this._config.chunkSize;
-		}
+		};
 
 		this._chunkLoaded = function()
 		{
-			if (xhr.readyState != 4)
+			if (xhr.readyState !== 4)
```

Comment:
If `readyState` were the string `"4"`, then this change would break. But I don't think that's possible.